Lombiq.HelpfulLibraries.LinqToDb/Docs/LinqToDb.md

```diff
@@ -0,0 +1,45 @@
+# Lombiq Linq2DB for Orchard Core

```

Comment:
Itt meg a manifestben is úgy kellene az elnevezés, ahogy itt le van írva: https://lombiq.atlassian.net/wiki/spaces/DEV/pages/19496967/Creating+a+new+Visual+Studio+project+or+solution